site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -43,6 +44,19 @@ const getConsequenceUrl = (type: Consequence["consequenceType"]) => {
 
 const ReviewDisruption = ({ disruption, previousSocialMediaPosts, csrfToken }: ReviewDisruptionProps): ReactElement => {
     const hasInitialised = useRef(false);
+    const [popUpState, setPopUpState] = useState<{ disruptionName: string; disruptionId: string }>();
+
+    const deleteActionHandler = (id: string, name: string): void => {
+        setPopUpState({ disruptionId: id, disruptionName: name });
+    };
+
+    const cancelActionHandler = (): void => {
+        setPopUpState(undefined);
+    };
+
+    const buildDeleteUrl = (idToDelete: string, csrfToken: string): string => {
+        return `/api/delete-disruption?id=${idToDelete}&_csrf=${csrfToken}`;
```

Comment:
Is there a reason for passing the csrf token through here rather than using the CsrfForm? Also it might be better to pass the disruptionId through as a hidden input in the form to keep it consistent with what we're doing elsewhere on the site

Reply:
i was doing it how fares does it but i can change the approach to this

Reply:
done